### PR TITLE
Add Authentik collection to hub

### PR DESCRIPTION
**Pull request by afirix**
Adding a collection, a parser, and a scenario for Authentik (https://goauthentik.io/).

### CODE REVIEW

File: parsers/s01-parse/firix/authentik-logs.yaml
Comment (LaurenceJJones) [inline parsers/s01-parse/firix/authentik-logs.yaml:6]: Could we use `evt.Parsed.message` instead of raw? sometimes, user want to log to another engine like syslog file this would cause an issue.

**Review by LaurenceJJones — CHANGES_REQUESTED** [review]
Request, one change to rename all `evt.Line.Raw` to `evt.Parsed.message` incase people use a logging service that doesnt pre treat the line. Once that is done, then all ready for merge

**Comment by afirix** [comment]
Done as requested. Tests still pass and all the functionality seem to be working as before the change.

**Comment by LaurenceJJones** [comment]
> Done as requested. Tests still pass and all the functionality seem to be working as before the change.

Yeah, it won't change anything on the surface, but if they use any logging facility

Waiting for automatic tests, then I will merge

**Review by LaurenceJJones — APPROVED** [review]
LGTM, thank you for your contribution to the CrowdSec community

**Pull request merged**
